lib/versioned/test.js

```diff
@@ -80,6 +80,7 @@ function Test(directory, pkgVersions, opts = {}) {
   this.duration = 0
   this.allPkgs = !!allPkgs
   this.strict = !!opts.strict
+  this.type = pkg.type
```

Comment:
nice

Reply:
Yeah--it turns out it was just *there*, so why not use it?